lang/de.json

```diff
@@ -715,38 +849,59 @@
     "Login failed": "Anmeldung fehlgeschlagen",
     "Login link sent, check your inbox": "Anmelde-Link gesendet, bitte überprüfen Sie Ihr Postfach",
     "Login name": "Login-name",
+    "Logistic Note": "Logistische Notiz",
     "Logistics note": "Logistik-Notiz",
     "Logo small": "Logo (klein)",
     "Logout": "Abmelden",
+    "Longitude": "Längengrad",
     "Mail": "Email",
     "Mail Account": "E-Mail-Konto",
+    "Mail Account Id": "E-Mail-Konto-ID",
     "Mail Accounts": "E-Mail-Konten",
     "Mail Body": "E-Mail-Text",
+    "Mail Cc": "CC",
     "Mail Folder": "E-Mail-Ordner",
+    "Mail Folder Id": "E-Mail-Ordner-ID",
     "Mail Messages": "E-Mail-Nachrichten",
     "Mail Subject": "E-Mail-Betreff",
+    "Mail To": "E-Mail an",
+    "Mailbox": "Postfach",
+    "Mailbox City": "Postfach Stadt",
+    "Mailbox Zip": "Postfach PLZ",
     "Main address": "Hauptadresse",
     "Main Address": "Hauptadresse",
     "Main product files": "Hauptproduktdateien",
     "Manage discount group": "Rabattgruppe verwalten",
     "Manage Logos": "Logos verwalten",
     "Manage users": "Benutzer verwalten",
     "Mandate Reference Number": "Mandatsreferenz",
+    "Manufacturer Product Number": "Hersteller-Produktnummer",
     "Manufacturer product number": "Herstellerproduktnummer",
```

Comment:
keep these two consistent. Although I don't think this even exists, can probably be removed.